extensions/industry/healthcare/fhir/fieldgroups/patient.schema.json

```diff
@@ -0,0 +1,241 @@
+{
+   "meta:license":[
+      "Copyright 2024 Adobe Systems Incorporated. All rights reserved.",
+      "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+      "you may not use this file except in compliance with the License. You may obtain a copy",
+      "of the License at https://creativecommons.org/licenses/by/4.0/"
+   ],
+   "$id":"https://ns.adobe.com/xdm/industry/healthcare/fhir/fieldgroup/patientdetail",
+   "$schema":"http://json-schema.org/draft-06/schema#",
+   "title":"Patient",
+   "type":"object",
+   "meta:tags":{
+      "industry":[
+         "healthcare_services"
+      ]
+   },
+   "meta:sensitive-data":true,
+   "description":"Demographics and other administrative details about an individual or animal receiving care or other health-related services.",
```

Comment:
We can add the line at the end o the description: "(Based on HL7 FHIR Release 5 specifications)"